src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -124,11 +130,64 @@ describe('Exam assessment', () => {
         });
     });
 
+    describe('Assess a quiz exercise submission', () => {
+        let resultDate: Dayjs;
+
+        beforeEach('Generate new exam name', () => {
+            examEnd = dayjs().add(15, 'seconds');
+            resultDate = examEnd.add(10, 'seconds');
+            prepareExam(examEnd, resultDate);
+        });
+
+        beforeEach('Create exercise and submission', () => {
+            courseManagementRequests.createQuizExercise({ exerciseGroup }, [multipleChoiceQuizTemplate], 'Cypress Quiz').then(() => {
+                courseManagementRequests.generateMissingIndividualExams(exam);
+                courseManagementRequests.prepareExerciseStartForExam(exam);
+                cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
+                examStartEnd.startExam();
+                cy.contains('Cypress Quiz').click();
+                cy.get('#answer-option-0 > .selection > .ng-fa-icon > .svg-inline--fa').click();
+                cy.get('#answer-option-2 > .selection > .ng-fa-icon > .svg-inline--fa').click();
+                cy.get('#exam-navigation-bar').find('.btn-danger').click();
+                examStartEnd.finishExam();
+            });
+        });
+
+        it('Assesses quiz automatically', () => {
+            cy.waitUntil(() => {
+                const now = dayjs().isAfter(examEnd);
+                if (now) {
+                    return now;
+                } else {
+                    cy.wait(1000);
+                }
+            }).then(() => {
```

Comment:
`waitUntil` seems to be kind of an overkill.
Is the following not possible?
```typescript
const now = dayjs();
if (now.isBefore(examEnd)) {
    cy.wait(now.diff(examEnd));
}
```
Also the cypress commands are queued so chaining isn't needed here if I understand correctly?